be/src/partnerWebhook/signature.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable new-cap */
```

Comment:
I wonder if we should start linking out to GitHub from the Developer Site once the churn dies down? Similar to GraphQL we could have the existing TypeScript example codeblock open this file in a new tab.